### DISCUSSION TITLE
Return Errors like API json error response

### DISCUSSION THREAD

**Discussion by basitalvi62** [spatie/laravel-webhook-client#146] [Q&A]
How can return custom json response from App\Handler\CustomSignatureValidator::class if laravel request validations are failed.

**Comment by wisliy** [comment]
Hey @basitalvi62 did you found a solution to this?

**Comment by majsterkoo** [comment]
For custom handling, I simply added a condition to Exceptions\Handler.php...

```php
if ($exception instanceof \Spatie\WebhookClient\Exceptions\InvalidWebhookSignature) {
    return response('', 401);
}
```
